PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,3 @@
+- [ ] Run `pod install` to ensure that the latest changes are in the Example project. Without this, Carthage might not see the latest changes.
```

Comment:
I think this needs a line about adding new files to pods and the main framework as that's tripped Anna and myself up in the past few weeks

Reply:
Updated